test/tests/functional/pbs_checkpoint.py

```diff
@@ -55,9 +55,12 @@ def setUp(self):
 kill $1
 exit 0
 """
-        self.abort_file = self.du.create_temp_file(body=abort_script)
-        self.du.chmod(path=self.abort_file, mode=0o755)
-        self.du.chown(path=self.abort_file, uid=0, gid=0, runas=ROOT_USER)
+        self.abort_file = self.du.create_temp_file(hostname=self.mom.hostname,
+                                                   body=abort_script)
+        self.du.chmod(hostname=self.mom.hostname,
+                      path=self.abort_file, mode=0o755)
+        self.du.chown(hostname=self.mom.hostname, path=self.abort_file,
+                      uid=0, gid=0, runas=ROOT_USER)
```

Comment:
Since this is a common mistake, maybe we should create a function as part of the mom similar to the function that creates a mom dynamic resource.  This one will create the checkpoint script, chmod/chown it, and set preempt_prio on the server.  This way people can always use this instead of this mistake popping up from time to time.

Reply:
@bhroam check the new commit.